gorush/notification_apns.go

```diff
@@ -179,8 +179,12 @@ func GetIOSNotification(req PushNotification) *apns2.Notification {
 		notification.Expiration = time.Unix(req.Expiration, 0)
 	}
 
-	if len(req.Priority) > 0 && req.Priority == "normal" {
-		notification.Priority = apns2.PriorityLow
+	if len(req.Priority) > 0 {
+		if req.Priority == "normal" {
+			notification.Priority = apns2.PriorityLow
+		} else if req.Priority == "high" {
+			notification.Priority = apns2.PriorityHigh
+		}
```

Comment:
Why we need another condition `req.Priority == "high"`?

You can set any value or ignore the property if you want the priority to `high`.

Reply:
Sorry only seeing your comment now, I must have missed it.

You are correct in that you could get away without the second condition. However I just prefer to be explicit about such things. Im not a fan of having API's "assume" things, hence why Id like to be explicit with high priority in the first place.

For example If a dev was to mistakenly put "medium" priority into the request, it would come out as "high", this could be confusing. Ideally in this instance gorush would return an error. The 2nd best option is to not set the property at all (and rely on the implicit default of APS), The third, would be to fall through and set the priority as high. IMO.

This also has the advantage of being "backwards compatible", in that people who never passed an explicit priority wont now be explicitly passing "high" to APS.

Im happy to change it, if its gonna block the PR.

Reply:
Agree totally